model/fv_arrays.F90

```diff
@@ -1269,6 +1269,8 @@ module fv_arrays_mod
     real, _ALLOCATABLE :: sgh(:,:)      _NULL  !< Terrain standard deviation
     real, _ALLOCATABLE :: oro(:,:)      _NULL  !< land fraction (1: all land; 0: all water)
     real, _ALLOCATABLE :: ts(:,:)       _NULL  !< skin temperature (sst) from NCEP/GFS (K) -- tile
+    real, _ALLOCATABLE :: usfco(:,:)    _NULL  !< sea surface current (zonal) from NCEP/GFS (m s-1) -- tile
+    real, _ALLOCATABLE :: vsfco(:,:)    _NULL  !< sea surface current (meridional) from NCEP/GFS (m s-1) -- tile
```

Comment:
Are these necessary?  I don't see them being used in your feature branch.

Reply:
@bensonr The variables usfco and vsfco are used in the updated surface layer scheme in ccpp-physics. The updated surface layer scheme is used in both parent and nested grids,

Reply:
@bensonr I think this is a good catch. The two lines of defining usfco/vsfco are not needed here inside the FV3dycore. The two variables are needed in the FV3ATM level though (including inside the moving_nest dir). @binli2337 is currently double checking and testing, will reply to this thread once he confirms. Thanks!